### ISSUE TITLE
[Bug]: Themes don't work correctly in 'classic' themes

### ISSUE THREAD

**Issue by Wagnerp** [Krypton-Suite/Standard-Toolkit#1271]
If a ribbon is being used in Windows 7 with the classic theme applied, it'll result with the following:

![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/949607/7e6b04c7-7f23-47f2-8bb8-7d9ae6d3e7e5)

**Comment by Smurf-IV** [comment]
Please retest now that #1269 is completed and merged into alpha

**Comment by Wagnerp** [comment]
As of 28/01/2024

![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/949607/f70a5590-1457-40f8-8683-a2136c5fa1c4)

**Comment by Smurf-IV** [comment]
re: the bottom image
- Is that an image capture artifact ?
- And if not - What OS with what SP?

re: top Image
- Seems like I left a bit of code change in the last PR, as the Outlook should have minimize and maximize visible
- I'll go check now.

**Comment by Smurf-IV** [comment]
Hmm.. Visible in Win 10
![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/2418812/23429983-ab23-42b4-b74a-797d4eaef034)

What os (and sp) is image 1 running on ?

**Comment by Wagnerp** [comment]
> Is that an image capture artifact ?
Unfortunately, no it happens when you switch themes

>  And if not - What OS with what SP?
Windows 7 64bit, SP1 using Aero/classic

**Comment by Wagnerp** [comment]
@Smurf-IV & @giduac

Is this worth keeping open since Win 7 has been EoL since 2014?

**Comment by Smurf-IV** [comment]
> Is this worth keeping open since Win 7 has been EoL since 2014?

Yes, As there are still a lot of POS and Manufacturing machines using Win 7. Also this needs to be tested on Win 8.1 as that is also used for Manufacturing machines.
Also as long as .net 4.8 is supported by MS, then keep win7 in the list.

**Comment by Smurf-IV** [comment]
In todays, Alpha (i.e. V90) then it works as expected:
![image](https://github.com/user-attachments/assets/c097b11f-3f65-41c0-b116-680d8171d115)
Note: the colours do not match those of the Office pictures in the OP

Removing V90

**Comment by Smurf-IV** [comment]
2007 blue is more like the OP Office tho ;-)
![image](https://github.com/user-attachments/assets/57ebee53-9f90-4d71-8d54-782f3ae442f6)

**Comment by Wagnerp** [comment]
@Smurf-IV 

Why are there little black triangles in the corners?

**Comment by Smurf-IV** [comment]
> @Smurf-IV
> 
> Why are there little black triangles in the corners?

That has always been the case for rounding on the main form (As far as I remember), when the Background theme of Windows is black, and you are drawing over another app.

**Comment by Smurf-IV** [comment]
"Classic" themes appear to work correctly now in V85:
![image](https://github.com/user-attachments/assets/ae1a9185-83b6-4c88-9d87-d34bb8e6b85f)
